package.json

```diff
@@ -49,6 +49,7 @@
     "express-handlebars": "3.0.0",
     "fs-extra": "2.1.2",
     "irc-framework": "2.8.0",
+    "jquery-textcomplete": "1.8.0",
```

Comment:
I mean, our definition of what is dev dependency and which is dependency is dodgy anyway, as you need the dependency to run the build, and we don't provide the build anywhere, so even running in prod, you'll still need to install the dev dependencies. But whatever makes you happy :-P